shell/platform/common/accessibility_bridge.cc

```diff
@@ -36,11 +36,26 @@ AccessibilityBridge::~AccessibilityBridge() {
   tree_->RemoveObserver(static_cast<ui::AXTreeObserver*>(this));
 }
 
+void AccessibilityBridge::AddFlutterSemanticsNodeUpdate(
+    const FlutterSemanticsNode2& node) {
+  pending_semantics_node_updates_[node.id] = FromFlutterSemanticsNode(node);
+}
+
+void AccessibilityBridge::AddFlutterSemanticsCustomActionUpdate(
+    const FlutterSemanticsCustomAction2& action) {
+  pending_semantics_custom_action_updates_[action.id] =
+      FromFlutterSemanticsCustomAction(action);
+}
+
+// TODO(loicsharma): Remove this as FlutterSemanticsNode is deprecated.
```

Comment:
I don't think we have other customer relying on this, what's preventing us from removing it?

Reply:
oh the migration of embedding will be a separate pr?

Reply:
Yup, Windows and macOS will be in subsequent pull requests. I will remove this once all affected embedders have been updated.